Documentation~/RuleOverrideTile.md

```diff
@@ -2,25 +2,25 @@
 
 __Contributions by:__ [johnsoncodehk](https://github.com/johnsoncodehk), [Autofire](https://github.com/Autofire)
 
-__Rule Override Tiles__ are Tiles which can override a subset of Rules for a given [Rule Tile](RuleTile.md) while keeping most of the others Rules originally set in the Rule Tile. This allows you to create Tiles that provide specialised behavior.
+__Rule Override Tiles__ are Tiles which can override a subset of Rules for a given [Rule Tile](RuleTile.md) while maintaining most of the other set Rules of the Rule Tile. This allows you to create Tiles that provide specialized behavior in specific scenarios.
 
 ## Properties
 
 | Property          | Function                                                     |
 | ----------------- | ------------------------------------------------------------ |
 | __Tile__          | The Rule Tile to override.                                   |
-| __Override Self__ | Enable this so that this Tile only accepts instances of itself when matching Rules. |
+| __Override Self__ | Enable this to have this Tile only accept instances of itself when matching Rules. |
```

Comment:
__Override Self__ option is no longer exists. And the advanced mode is move to __AdvancedRuleOverrideTile__.

Reply:
Thanks! Will update this too!

Reply:
I changed the documentation for this and added one for the AdvancedRuleOverrideTile.

Reply:
nice job!